laia/scripts/htr/create_model.py

```diff
@@ -49,6 +49,11 @@ def run(
         "Model has {} parameters",
         sum(param.numel() for param in model.parameters()),
     )
+    if fixed_input_height:
+        log.info(
+            f"Images should be at least {model.get_min_valid_image_size()} pixels wide for convolutions. "
+            f"Smaller images will be padded."
+        )
```

Comment:
We should not compute this value when creating the model but search for it when we start the training. The range of values to test depends heavily on the size of the images used during training.

Reply:
This value is now only calculated before training the model.